packages/containers/src/DeleteResource/DeleteResource.container.js

```diff
@@ -2,15 +2,18 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { componentState } from '@talend/react-cmf';
 import { ConfirmDialog } from '@talend/react-components';
+import { translate, Trans } from 'react-i18next'; // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
You should not disable this rule.
It highlight an issue in your dependencies. You have (react-)i18next as dev deps, but that's wrong. You should either add it as peer dependencies or move to to dependencies.